inc/byline_class.php

```diff
@@ -103,7 +103,7 @@ function avatar() {
 		} else {
 			$author_email = get_the_author_meta( 'email', $this->author_id );
 			if ( $this->author->type == 'guest-author' && get_the_post_thumbnail( $this->author->ID ) ) {
-				$output = get_the_post_thumbnail( $this->author->ID, array( 32,32 ) );
+				$output = get_the_post_thumbnail( $this->author->ID, array( 60,60 ) );
```

Comment:
32x32 image size doesn't exist in Largo, 60x60 thumbnails are defined in 
https://github.com/INN/Largo/blob/56c40405990d7693fadfa082196b5339963d445b/functions.php#L256